lib/active_merchant/billing/gateways/global_collect.rb

```diff
@@ -289,8 +291,10 @@ def error_code_from(succeeded, response)
         unless succeeded
           if errors = response["errors"]
             errors.first.try(:[], "code")
+          elsif status = response["statusOutput"]["statusCode"]
```

Comment:
I'm not sure I see how it would hit an exception in that case. Won't it just kick to `else "No error code available"` if there's no `statusCode`? Testing seems to confirm that.